.python-version

```diff
@@ -1 +1 @@
-3.8.2
+3.8.3
```

Comment:
This needs removing given it's unavailable in pyenv at the moment

Reply:
Going back on this given it doesn't appear quite as simple as changing a tag: https://github.com/ONSdigital/eq-questionnaire-runner/pull/129